src/widgets/histogram/histogram-title-template.tpl

```diff
@@ -1,16 +1,18 @@
-<h3 class="CDB-Text CDB-Size-large is-overflow" title="<%- title %>"><%- title %></h3>
-<div class="CDB-Widget-options CDB-Widget-contentSpaced">
-  <button class="CDB-Widget-buttonIcon CDB-Widget-buttonIcon--circle js-sizes
-    <%- isSizesApplied ? 'is-selected' : '' %>
-    <%- isSizesApplied ? 'js-cancelSizes' : 'js-applySizes' %>
-    " data-tooltip="<%- isSizesApplied ? 'Remove sizes' : 'Apply sizes' %>">
-    <i class="CDB-IconFont CDB-IconFont-drop CDB-IconFont--small CDB-IconFont--top"></i>
-  </button>
-  <button class="CDB-Shape js-actions u-lSpace">
-    <div class="CDB-Shape-threePoints is-blue is-small">
-      <div class="CDB-Shape-threePointsItem"></div>
-      <div class="CDB-Shape-threePointsItem"></div>
-      <div class="CDB-Shape-threePointsItem"></div>
-    </div>
-  </button>
+<div class="CDB-Widget-title CDB-Widget-contentSpaced">
```

Comment:
I wrap into that element because initially it was wrapper on it. See: https://github.com/CartoDB/deep-insights.js/blob/master/src/widgets/histogram/content.tpl#L3